apollo-backend/src/main/java/io/logz/apollo/controllers/DeploymentController.java

```diff
@@ -81,19 +86,33 @@ public String getDeploymentEnvStatus(int id) {
 
     @LoggedIn
     @POST("/deployment")
-    public void addDeployment(int environmentId, int serviceId, int deployableVersionId, String deploymentMessage, Req req) {
-        try {
-            Deployment deployment = deploymentHandler.addDeployment(environmentId, serviceId, deployableVersionId, deploymentMessage, req);
-            assignJsonResponseToReq(req, HttpStatus.CREATED, deployment);
-        } catch (ApolloDeploymentBlockedException e) {
-            assignJsonResponseToReq(req, HttpStatus.FORBIDDEN, e.getMessage());
-        } catch (ApolloDeploymentConflictException e) {
-            assignJsonResponseToReq(req, HttpStatus.CONFLICT, e.getMessage());
-        } catch (ApolloDeploymentTooManyRequestsException e) {
-            assignJsonResponseToReq(req, HttpStatus.NOT_ACCEPTABLE, e.getMessage());
-        } catch (Exception e) {
-            assignJsonResponseToReq(req, HttpStatus.INTERNAL_SERVER_ERROR, e.getMessage());
-        }
+    public void addDeployment(String environmentIdsCsv, String serviceIdsCsv, int deployableVersionId, String deploymentMessage, Req req) {
```

Comment:
This is still OK now, but please keep in mind that this is now an opensource project and we cant just change signatures of our rest endpoints. Lets please also document that in a "release notes" document in the wiki that indicates the naming change

Reply:
And make sure that all documentation on that API is changes as well